.gitignore

```
@@ -0,0 +1,2 @@
+node_module

```

this should be `node_modules`
```suggestion
node_modules

```